sa/model.go

```diff
@@ -418,6 +418,7 @@ func statusUint(status core.AcmeStatus) uint8 {
 	return statusToUint[string(status)]
 }
 
+// Changes to this string must be carried through to sa.NewAuthorizations2
```

Comment:
Nit, comments for a constant should begin with the name of the constant as the subject of a complete sentence ending in punctuation.

```go
// authzFields ...text here.
```

Reply:
```
// authzFields is used in a variety of places in sa.go, and modifications to
// it must be carried through to every use in sa.go
```